pkg/pillar/types/dns.go

```diff
@@ -35,6 +35,7 @@ type NetworkPortStatus struct {
 	Alias          string // From SystemAdapter's alias
 	IsMgmt         bool   // Used to talk to controller
 	IsL3Port       bool   // True if port is applicable to operate on the network layer
+	InvalidConfig  bool   // True if port has invalid config and should not be used
```

Comment:
"invalid config" might be a bit too vague - isn'this basically about parse errors (such as invalid IP address syntax etc - I haven't looked at all what the parser checks.)
"invalid" could mean that the proxy config doesn't work, or that a static IP does not match the submet(s) used on the link, or other invalid things which cause unreachability.

So would it be more clear to call this "incorrect port config syntax"? Or does the parser check more than just syntax?

Reply:
It covers all static checks: parsing error, missing something, IP not inside a subnet, etc.
Basically "invalid" means that the config cannot be even applied.

Reply:
But "invalid" doesn't mean "unapplicable"  based on my dictionary.

Reply:
"Invalid" is quite commonly used for network config that failed parsing or (static) validation checks.
Another adjective I can think of is "malformed".
I would like to keep it one-word for brevity.

Reply:
OK, let's keep the name and add something like "parsing error, missing something, IP not inside a subnet, etc." to the comment.

Reply:
Done